db/migrate/20181029170924_force_update_broken_manual_published.rb

```diff
@@ -0,0 +1,14 @@
+class ForceUpdateBrokenManualPublished < Mongoid::Migration
+  def self.up
+    user = User.find_by(email: "oscar.wyatt@digital.cabinet-office.gov.uk")
+    if user
+      manual = Manual.find("b59056ac-f7e9-4415-96b5-79cc5cfb0a76", user)
+      if manual
+        manual.editions.sort_by{|ed| ed.created_at}.last.update(state: "published")
```

Comment:
I think you could just use `last` here without the `sort_by` but otherwise looks good.